distribution/pom.xml

```diff
@@ -481,6 +482,7 @@
                 <configuration>
                   <buildArgs>
                     <VERSION>${project.version}</VERSION>
```

Comment:
@xvrl : Is there a way we can pass this from `cc-druid`? Or is this the right place to pass it? Because in cc-druid I see that we are only invoking the `mvn` command [here](https://github.com/confluentinc/cc-druid/blob/master/Makefile#L48)